python/lsst/obs/decam/ingestCalibs.py

```diff
@@ -105,3 +82,73 @@ def getExtensionName(md):
         @param md (PropertySet) FITS header metadata
         """
         return md.get('EXTNAME')
+
+    def getDestination(self, butler, info, filename):
+        """Get destination for the file
+
+        @param butler      Data butler
+        @param info        File properties, used as dataId for the butler
+        @param filename    Input filename
+        @return Destination filename
+        """
+        # Arbitrarily set ccdnum = 1 to make the mapper template happy
+        info["ccdnum"] = 1
+        if "flat" in info.get("obstype").strip().lower():
+            raw = butler.get("cpFlat_filename", info)[0]
+        elif info.get("obstype").strip().lower() == "zero":
+            raw = butler.get("cpBias_filename", info)[0]
+        else:
+            assert False, "Invalid OBSTYPE '{:s}'".format(info.get("obstype"))
+        # Remove HDU extension (ccdnum) since we want to refer to the whole file
+        c = raw.find("[")
+        if c > 0:
+            raw = raw[:c]
+        return raw
+
+
+class DecamIngestCalibsArgumentParser(IngestCalibsArgumentParser):
+    """Argument parser to support ingesting calibration images into the repository"""
+    def __init__(self, *args, **kwargs):
+        IngestCalibsArgumentParser.__init__(self, *args, **kwargs)
+        self.add_argument("--mode", choices=["move", "copy", "link", "skip"], default="link",
+                          help="Mode of delivering the files to their destination")
+
+
+class DecamIngestCalibsConfig(IngestCalibsConfig):
+    """Configuration for DecamIngestCalibsTask"""
+    allowError = Field(dtype=bool, default=False, doc="Allow error in ingestion?")
+    clobber = Field(dtype=bool, default=False, doc="Clobber existing file?")
+
+
+class DecamIngestCalibsTask(IngestCalibsTask):
```

Comment:
I'm happy to do that, but since it would change the behavior of how all calibrations are ingested (symlinking the calib files) beyond just obs_decam, is this the kind of thing that should go through an RFC first?